Rakefile

```diff
@@ -18,6 +18,51 @@ task :check_test_env do
   raise 'RAILS_ENV not set to test' unless Rails.env.test?
 end
 
+task :build_indexes do
+  require 'scalarm/service_core/initializers/mongo_active_record_initializer'
+
+  MongoActiveRecordInitializer.start(Rails.application.secrets.database) unless Rails.env.test?
+
+  mongo_active_records = Scalarm::Database::MongoActiveRecord.descendants.select do |c|
+    not (c.name.include?("CappedMongoActiveRecord") or c.name.include?("EncryptedMongoActiveRecord"))
+  end.map{|c| c.name}.compact.map do |c_name|
+    Object.const_get(c_name)
+  end
+
+  mongo_active_records.each do |mar|
+    puts "Processing '#{mar.name}' - #{mar._indexed_attributes} ..."
+    record_collection = mar.collection
+
+    mar._indexed_attributes.each do |attr|
+      puts "Checking if an index for '#{attr}' exists ..."
+      index_exist = false
+
+      record_collection.index_information.each do |_, index_info|
```

Comment:
Pomysł dobry ale wysoka wydajność nie jest tutaj potrzebna. Ten kod bedzie wykonywany na zadanie w trybie offline i to w przypadkach gdy baza ulegnie zmianie. Czytelność kodu jest tutaj ważniejsza.